### PR TITLE
Add content from CAS About Us

### PR DESCRIPTION
**Pull request by diatomsRcool**
added team.html and collaborators.html and students.html

### CODE REVIEW

File: docs/team.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE HTML>
+<!--
+	Theory by TEMPLATED
+	templated.co @templatedco
+	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
+-->
+<html>
+	<head>
+		<title>Our Team</title>
```
Comment (monicacecilia) [inline docs/team.html:9]: There are 5 or 6 different types of circles around these photos. I don't know what the cause is, and I don't think I want us to focus on that right now, but if we can 'prettify' later, that would be good.  😬

File: docs/team.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            <div class="image round"><a href="awalden.html">
+              <img src="images/awalden.jpg" alt="Anita Walden" />
+            </div>
+            <h3><a href="awalden.html">Anita Walden</a></h3>
+            <p>Assistant Director, CD2H</p>
+          </div>
+
+          <div class="box person">
+            <div class="image round"><a href="bwhite.html">
+              <img src="images/brandon.jpg" alt="Brandon White" />
```
Comment (monicacecilia) [inline docs/team.html:196]: this image file does not exist in https://github.com/tis-lab/tislab.org/tree/CASabout/docs/images

File: docs/team.html (hunk 1 above)
Comment (monicacecilia) [reply docs/team.html:9]: For example, there are 4 different kinds of 'rounds' here. 😉 

Screen Shot 2021-03-02 at 5 05 43 PM

**Review by monicacecilia — CHANGES_REQUESTED** [review]
See comment. We may need to add some place holder image for Brandon.

File: docs/team.html (hunk 1 above)
Comment (diatomsRcool) [reply docs/team.html:9]: added

File: docs/team.html (hunk 2 above)
Comment (diatomsRcool) [reply docs/team.html]: I switched it to the placeholder image

Comment (diatomsRcool) [reply docs/team.html]: I switched this to the generic image file

File: docs/team.html (hunk 1 above)
Comment (diatomsRcool) [reply docs/team.html:9]: This is a feature of image.round is the css. The rounding is automatically done based on the type of image. To fix this we would have to be stricter about getting more standardized "mug shots" from everyone.

Comment (diatomsRcool) [reply docs/team.html:9]: I made a small change that I think might help, but we will have to revisit this later.

**Pull request merged**